### PR TITLE
Week 9 assignment

### PR DESCRIPTION
**Pull request by sachin286**

### CODE REVIEW

File: Week-9/todolist/src/App.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  const markTodo = (index) => {
+    const newTodos = [...todos];
+    console.log("making done item", newTodos[index]);
+    newTodos[index].isDone = true;
+    setTodos(newTodos);
+  };
+
+  const removeTodo = (index) => {
+    const newTodos = [...todos];
+    console.log(newTodos);
```
Comment (ShashankSuresh) [inline Week-9/todolist/src/App.js]: Please remove `console.log` statements

File: Week-9/todolist/src/App.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      </div>
+      <button className="button blue" type="submit">
+        Submit
+      </button>
+    </form>
+  );
+}
+
+function App() {
+  const [todos, setTodos] = React.useState([
```
Comment (ShashankSuresh) [inline Week-9/todolist/src/App.js]: `import React, { useState } `

The above is a better way of including hooks than `React.useState`

File: Week-9/todolist/src/App.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+          onClick={() => removeTodo(index)}
+        >
+          Delete
+        </button>
+      </div>
+    </div>
+  );
+}
+
+function FormTodo({ addTodo }) {
```
Comment (ShashankSuresh) [inline Week-9/todolist/src/App.js]: This could be a separate component.

File: Week-9/todolist/src/App.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import './App.css';
+import "bootstrap/dist/css/bootstrap.min.css";
+
+function Todo({ todo, index, markTodo, removeTodo }) {
```
Comment (ShashankSuresh) [inline Week-9/todolist/src/App.js]: This could be a separate component.

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Comment by ShashankSuresh** [comment]
@sachin286 Please attach screenshots of the app.

**Comment by ShashankSuresh** [comment]
image

Im able to add an empty todo item.

**Comment by ShashankSuresh** [comment]
image
Please add a key value to all elements which are constructed using a loop.

**Comment by sachin286** [comment]
updated the codebase as suggested

**Review by ShashankSuresh — APPROVED** [review]

**Pull request merged**